app/components/Calendar.tsx

```diff
@@ -5,27 +5,81 @@ import EventModal from '../components/common/eventModal';
 import RdsCalendar from '../components/common/rdsCalendar';
 // tilde operator not working so used relative imports
 
-const initialEventsList: CalEvent[] = [
+export const initialEventsList: CalEvent[] = [
```

Comment:
@prerana1821 can we move this constant variable into another file like `components/Calendar.constants.ts` and change the variable name to `INITIAL_EVENTS_LIST` from `initialEventList` as this is a constant value and it's a common practice to name them in the capital?

Reply:
@yesyash there is already an issue created to work on it.

Reply:
@iRohitGaur can you please link it here and resolve this?